frontend/src/data/config/FakeDatasetMetadata.ts

```diff
@@ -107,6 +102,21 @@ const datasetMetadataList: DatasetMetadata[] = [
     name: "COVID-19 deaths, cases, and hospitalizations by gender and state",
     update_time: "March 2021",
   },
+  {
+    id: "uhc_age",
+    name: "COPD and Diabetes prevelence by age and state",
+    update_time: "2020",
```

Comment:
Just confirming, this update time is correct, right? ie the united health foundation website uses BRFSS data from 2020?

Reply:
Should be 2019, my b